CODEOWNERS

```diff
@@ -1,11 +1,12 @@
-# Below is a list of Flutter team members who are suggested reviewers
-# for contributions to this repository.
+# Below is a list of Flutter team members' Github handles who are
+# suggested reviewers for contributions to this repository.
 #
 # These names are just suggestions. It is fine to have your changes
 # reviewed by someone else.
 #
 # Use git ls-files '<pattern>' without a / prefix to see the list of matching files.
 
-/CODEOWNERS @magder
-/packages/flutter_tools/templates/module/ios/ @magder
-/packages/flutter_tools/templates/**/Podfile* @magder
+/bin/internal/goldens.version @Piinks
```

Comment:
Do you want to include /dev/bots/download_goldctl.sh, /packages/flutter_goldens, etc?  Or will this goldens.version always be bumped with those changes?

Reply:
For now, it'll always bump. :) 
I would like to keep a close eye on it while changing over to the new system that won't use this anymore.